pom.xml

```diff
@@ -36,6 +29,15 @@
     <tag>ws-cleanup-0.30</tag>
```

Comment:
Shouldn't be there finally ws-cleanup-0.34 as the last released version?

Reply:
I already commented about this here https://github.com/jenkinsci/ws-cleanup-plugin/commit/77d4a367c7611183fa697024ece6d2f997d7aedd#diff-600376dffeb79835ede4a0b285078036.
Changing values is out of scope for this commit. It is just reformatting the `pom.xml` with `mvn tidy:pom`.